Atividades/Em casa/atividade.js

```diff
@@ -0,0 +1,233 @@
+class HistoricoMedico {
+  constructor() {
+    this.consultas = [];
+    this.vacinas = [];
+    this.procedimento = [];
+  }
+}
+
+class Animal {
+  constructor(nome, idade, cor, consultas = []) {
+    this.nome = nome;
+    this.idade = idade;
+    this.cor = cor;
+    this.historico = new HistoricoMedico();
+  }
+
+  consultar() {
+    const dataConsulta = new Date();
+    this.consultas.push(dataConsulta);
+    this.historico.consultas.push(dataConsulta);
+  }
```

Comment:
aqui em consultar poderia ser um pouco mais conciso e sem precisar criar uma const
``` 
consultar() {
      this.historico.consultas.push(Date());
    }
 ```
 
 e o "this.consultas.push(dataConsulta);" o meu console relacmou disso e se tirar o resto tb funcionaria!